app/main.py

```diff
@@ -2,0 +1,28 @@
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split(" ")) != 3 or command.split(" ")[0] != "mv":
+        print("wrong command")
+        return
+
+    file_name, new_path = command.split(" ")[1:]
+
+    with open(file_name, "r") as sorce:
+        text = sorce.read()
+
+    os.remove(file_name)
+    dir_list = new_path.split("/")
+    directory = ""
+
+    for index in range(len(dir_list) - 1):
+        if index:
+            directory += "/" + dir_list[index]
+        else:
+            directory += dir_list[index]
+        if os.path.exists(directory):
+            continue
+        os.mkdir(directory)
+
+    with open(new_path, "w") as new_file:
```

Comment:
The code does not handle cases where the destination path ends with a `/`. Ensure that the file name is correctly extracted and used.